oxenstored/xenstored.ml

```diff
@@ -28,21 +28,23 @@ let debug fmt = Logging.debug "xenstored" fmt
 
 let info fmt = Logging.info "xenstored" fmt
 
+let spec_fds = ref []
```

Comment:
This is only ever written to once, it could be passed as a parameter where needed by adding a new parameter to `process_connection_fds`.